components/cqrs/marshaler_json.go

```diff
@@ -0,0 +1,48 @@
+package cqrs
+
+import (
+	"encoding/json"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill/message"
+)
+
+type JsonMarshaler struct {
```

Comment:
`JSONMarshaler`

strange that the linter didn't catch this

Reply:
renamed, 7570713
I think that this linter doesn't care about it